README.md

```diff
@@ -82,37 +138,112 @@ The two tools can be built and used independently.
+variable. If you can run the following commands in a Windows cmd without
+entering your password, you are all set:
 ```
-Depending on your setup, you may have to specify additional arguments for the
-ssh and scp commands, including proper quoting, e.g.
+ssh user@linux.device.com
+scp somefile.txt user@linux.device.com:
 ```
-cdc_rsync --ssh-command="\"C:\path with space\to\ssh.exe\" -F ssh_config_file -i id_rsa_file -oStrictHostKeyChecking=yes -oUserKnownHostsFile=\"\"\"known_hosts_file\"\"\"" --scp-command="\"C:\path with space\to\scp.exe\" -F ssh_config_file -i id_rsa_file -oStrictHostKeyChecking=yes -oUserKnownHostsFile=\"\"\"known_hosts_file\"\"\"" C:\path\to\assets\* user@linux.machine.com:~/assets -vr
+Here, `user` is the Linux user and `linux.device.com` is the Linux host to
+SSH into resp. copy the file to.
+
+If `ssh.exe` or `scp.exe` cannot be found, or if additional arguments are
+required, it is recommended to set the environment variables `CDC_SSH_COMMAND`
+and `CDC_SCP_COMMAND`. The following example specifies a custom path to the SSH
+and SCP binaries, a custom SSH config file, a key file and a known hosts file:
 ```
-Lengthy ssh/scp commands that rarely change can also be put into environment
-variables `CDC_SSH_COMMAND` and `CDC_SCP_COMMAND`, e.g.
+set CDC_SSH_COMMAND="C:\path with space\to\ssh.exe" -F C:\path\to\ssh_config -i C:\path\to\id_rsa -oStrictHostKeyChecking=yes -oUserKnownHostsFile="""C:\path\to\known_hosts"""
+set CDC_SCP_COMMAND="C:\path with space\to\scp.exe" -F C:\path\to\ssh_config -i C:\path\to\id_rsa -oStrictHostKeyChecking=yes -oUserKnownHostsFile="""C:\path\to\known_hosts"""
 ```
-set CDC_SSH_COMMAND="C:\path with space\to\ssh.exe" -F ssh_config_file -i id_rsa_file -oStrictHostKeyChecking=yes -oUserKnownHostsFile="""known_hosts_file"""
 
-set CDC_SCP_COMMAND="C:\path with space\to\scp.exe" -F ssh_config_file -i id_rsa_file -oStrictHostKeyChecking=yes -oUserKnownHostsFile="""known_hosts_file"""
+#### Google Specific
+You can use these tools to sync or stream data from a gWindows device to a
+gLinux device. To set up SSH authentication, set the following environment
+variables:
+```
+set CDC_SSH_COMMAND=C:\gnubby\bin\ssh.exe
+set CDC_SCP_COMMAND=C:\gnubby\bin\scp.exe
+```
+Note that you will have to touch your security key multiple times during the
+first run. Subsequent runs only require a single touch.
 
-cdc_rsync C:\path\to\assets\* user@linux.machine.com:~/assets -vr
+### CDC RSync
+`cdc_rsync` is used similar to `scp` or the Linux `rsync`. To sync a single
+Windows file `C:\path\to\file.txt` to the home directory `~` on the Linux device
+`linux.device.com`, run
+```
+cdc_rsync C:\path\to\file.txt user@linux.device.com:~
+```
+`cdc_rsync` understands the usual Windows wildcards `*` and `?`.
+```
+cdc_rsync C:\path\to\*.txt user@linux.device.com:~
+```
+To sync the contents of the Windows directory `C:\path\to\assets` recursively to
```

Comment:
I'd like to keep this, in case someone "fast forwards" to cdc_stream and misses the fact that it streams from Windows to Linux (an the fact that the dir starts with C:).